openpype/hosts/maya/plugins/publish/collect_render.py

```diff
@@ -293,6 +293,7 @@ def process(self, context):
                 "source": filepath,
                 "expectedFiles": full_exp_files,
                 "publishRenderMetadataFolder": common_publish_meta_path,
+                "renderProducts": layer_render_products,
```

Comment:
Note that this is **not** JSON serializable and thus the logging below had to be removed to avoid an error. Is this a problem?